Notebooks/beavis-ci.csh

```diff
@@ -0,0 +1,168 @@
+#! /usr/bin/env tcsh
+#=======================================================================
+#+
+# NAME:
+#   beavis-ci.csh
+#
+# PURPOSE:
+#   Enable occasional integration and testing. Like travis-ci but dumber.
+#
+# COMMENTS:
+#   Makes "rendered" notebooks and deploys them to a "rendered" orphan 
+#   branch, pushed to GitHub for web display. 
+#
+# INPUTS:
+#
+# OPTIONAL INPUTS:
+#   -h --help     Print this header
+#   -u --username GITHUB_USERNAME, defaults to the environment variable
+#   -k --key      GITHUB_API_KEY, defaults to the environment variable
+#   -n --no-push  Only run the notebooks, don't deploy the outputs
+#   --html        Make html outputs instead
```

Comment:
Default operation is (now) to make rendered notebooks and push them to a branch called "rendered" (following @cwwalter 's suggestion).